warehouse/models/mart/gtfs/_mart_gtfs.yml

```diff
@@ -1292,3 +1306,58 @@ models:
         - not_null
       - *_valid_from
       - *_valid_to
+  - name: fct_daily_scheduled_service_summary
+    description: |
+      This is daily service totals by feed.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - service_date
+            - feed_key
+      - dbt_utils.expression_is_true:
+          expression: "ttl_service_hours >= 0"
+    columns:
+      - name: service_date
+        description: Date on which this service level was present
+        tests:
+          - not_null
+      - name: feed_key
+        tests:
+          - not_null
+        description: feed_key only present with active service on date
+      - name: ttl_service_hours
+        description: Total service hours across entire feed on date, if any
+      - name: n_trips
+        description: Total number of trips ran on date, if any
+      - name: first_departure_ts
+        description: Time of first departure on date
+      - name: last_arrival_ts
+        description: Time of last departure on date
+      - name: n_stop_times
+        description: |
+         Total number of stop events present in active feed
+         (not necessarily served on date)
+      - name: n_routes
+        description: |
+         Total number of routes that had trips scheduled on this day
+         (not necessarily operating on date)
+exposures:
+  - name: calitp_reports_site
```

Comment:
I would maybe
1. Move this to the `mart_gtfs_quality` YAML because reports is in the quality domain
2. Comment this section and remove the specific dependencies until the reports site is actually consuming this data -- so leave everything but the `depends_on` section but comment it out and put a `TODO` comment at the top indicating that once the reports site is actually migrated it should be filled out